handler.js

```diff
@@ -194,10 +194,11 @@ async function approve(responseUrl, emailAtt, user, submissionId) {
     let emailSubject = emailAtt.title;
     let emailBody = EMAIL_TEMPLATE(tmplContext);
     const ses = new AWS.SES();
-    await ses.sendEmail({
+    const emailOpts = {
```

Comment:
I broke this out into a separate variable so I could print it out and check to make sure that everything looked right before sending the email.

Reply:
The CC change sounds good but I think it's going to break the integration tests – have you run those yet?

Reply:
Just fixed them.